src/main/java/net/glowstone/entity/projectile/GlowWitherSkull.java

```diff
@@ -0,0 +1,22 @@
+package net.glowstone.entity.projectile;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.bukkit.Location;
+import org.bukkit.entity.WitherSkull;
+
+public class GlowWitherSkull extends GlowFireball implements WitherSkull {
+    private static final int NETWORK_ID = 66;
+    @Getter
+    @Setter
+    private boolean charged;
+
+    @Override
+    protected int getObjectId() {
+        return NETWORK_ID;
+    }
+
+    public GlowWitherSkull(Location location) {
```

Comment:
Put the constructor above other methods.

Reply:
Done in f97a894